src/store.js

```diff
@@ -199,6 +199,7 @@ export class Store {
     return this._helper.lastResults.processingTimeMS;
   }
 
+  // todo: change to goToFirstPage()
```

Comment:
BTW that is what Beta's are for, I doubt there are custom components relying on that un-documented method.